pkgs/applications/networking/cluster/openshift/default.nix

```diff
@@ -37,6 +37,18 @@ in stdenv.mkDerivation rec {
 
   patchPhase = ''
     patchShebangs ./hack
+
+    substituteInPlace pkg/oc/bootstrap/docker/host/host.go  \
+      --replace 'nsenter --mount=/rootfs/proc/1/ns/mnt findmnt' \
+      'nsenter --mount=/rootfs/proc/1/ns/mnt ${utillinux}/bin/findmnt'
+
+    substituteInPlace pkg/oc/bootstrap/docker/host/host.go  \
+      --replace 'nsenter --mount=/rootfs/proc/1/ns/mnt mount' \
+      'nsenter --mount=/rootfs/proc/1/ns/mnt ${utillinux}/bin/mount'
+
+    substituteInPlace pkg/oc/bootstrap/docker/host/host.go  \
+      --replace 'nsenter --mount=/rootfs/proc/1/ns/mnt mkdir' \
+      'nsenter --mount=/rootfs/proc/1/ns/mnt ${utillinux}/bin/mount'
```

Comment:
shouldn't it be `mkdir` instead of `mount` ?

Reply:
Thank for the review. I have accidentally overseen this. Fix to master: #41274